### PR TITLE
remove all F1 related contents

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Remove all F1 related content from docs-cn in v4.0 and v5.0. For other versions, @bb7133 will create a separate PR.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5352 https://github.com/pingcap/docs/pull/5346 https://github.com/pingcap/docs/pull/5423
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-computing.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ TiDB 中每个 `Database` 和 `Table` 都有元信息，也就是其定义以及
 
 每个 `Database`/`Table` 都被分配了一个唯一的 ID，这个 ID 作为唯一标识，并且在编码为 Key-Value 时，这个 ID 都会编码到 Key 中，再加上 `m_` 前缀。这样可以构造出一个 Key，Value 中存储的是序列化后的元信息。
 
-除此之外，TiDB 还用一个专门的 (Key, Value) 键值对存储当前所有表结构信息的最新版本号。这个键值对是全局的，每次 DDL 操作的状态改变时其版本号都会加 1。目前，TiDB 把这个键值对持久化存储在 PD Server 中，其 Key 是 "/tidb/ddl/global_schema_version"，Value 是类型为 int64 的版本号值。TiDB 参考了 Google F1 的 Online Schema 变更算法，有一个后台线程在不断地检查 PD Server 中存储的表结构信息的版本号是否发生变化，并且保证在一定时间内一定能够获取版本的变化。
+除此之外，TiDB 还用一个专门的 (Key, Value) 键值对存储当前所有表结构信息的最新版本号。这个键值对是全局的，每次 DDL 操作的状态改变时其版本号都会加 1。目前，TiDB 把这个键值对持久化存储在 PD Server 中，其 Key 是 "/tidb/ddl/global_schema_version"，Value 是类型为 int64 的版本号值。TiDB 采用 Online Schema 变更算法，有一个后台线程在不断地检查 PD Server 中存储的表结构信息的版本号是否发生变化，并且保证在一定时间内一定能够获取版本的变化。
```
Comment (CharLotteiu) [inline tidb-computing.md:115]: Is there a PR that aligns this change to the docs repo? Don't see the corresponding content in the two PR links listed in the PR description.

**Review by CharLotteiu — APPROVED** [review]
LGTM

File: tidb-computing.md (hunk 1 above)
Comment (TomShawn) [reply tidb-computing.md:115]: Done in https://github.com/pingcap/docs/pull/5423. PTAL, thanks!

**Pull request merged**